docs/misc/changelog.rst

```diff
@@ -5,6 +5,25 @@ Changelog
 
 For download links, please look at `Github release page <https://github.com/hill-a/stable-baselines/releases>`_.
 
+Pre-Release 2.10.3a0 (WIP)
```

Comment:
please update the version in the `version.txt` file too ;)

Reply:
Done!